package.json

```diff
@@ -54,7 +54,8 @@
     "semantic-release": "15.13.3"
   },
   "jest": {
-    "preset": "react-native"
+    "preset": "react-native",
+    "testMatch": [ "**/__tests__/**/*.[jt]s?(x)", "**/?(*.)+(test).[jt]s?(x)" ]
```

Comment:
this looks pretty much lika a default testMatch, do you really need to change it?. Also, format with Prettier

Reply:
Good point, I left `.spec` out, for e2e tests. Going to change that.